blog/content/post/2019-11-22-Websummit-2019.md

```diff
@@ -0,0 +1,107 @@
+
+"Tech event" is the keyword that turns all of us on and makes us travel over
+thousand kilometers, but after the struggle, I was the one who becomes the first
+official 3mdeb representative on that event. Full of hope and curiosity, with
+a great financial and organizational support provided by
+[Pomeranian Export Broker](http://www.brokereksportowy.pl), at Sunday, November
+3, I've joined Polish group on the Lech Walesa Airport in Gdansk.
+
+#### Have you heard about the main types of a lie?
+
+In the classical version, there are small lies, big lies, and statistics. I've
+found another - calling Web Summit a tech event. Blows up the scale. It was
+political, it was sociological, it was futuristic, but it has much more to say
+about the people relations, than about technology.
+
+The opening day was a series of presentations. The main guest wasn't physically
+with us, but his speech was streamed from Russia. Edward Snowden - the great
+warrior for freedom of speech, which had left the western world and settled down
+in the capital of freedom - Moscow. Though I don't consider him the most
+reliable man living, the speech itself was fine. In the end, he started to
+accuse companies by their name for trying to enslave the world population, and
+when he got to Huawei it appeared that he runs out of time. The next speaker was
+Guo Ping, Huawei's Rotating Chairman - astonishing coincidence, isn't it?
+He tried to persuade an audience that all the things anyone heard about his
+company are a lie, no matter the facts. Hopefully, there were a lot of young
+and inexperienced people there, in the other case, the audience might not be
+persuaded.
+
+After the opening, there should be a Night Summit event. But it was raining, and
+almost no one came.   
+
+Day two started with a something-like-fairs, but many of the companies
+exhibiting there had nothing but themselves for sale. At first, I stayed
+at [Polish Agency for Enterprise Development](https://en.parp.gov.pl/) stand.
+I want to sincerely thank all the people there for helping me to present our
+products and let me leave our materials on the stand. After several hours of
+explaining to children what firmware and embedded are, I've decided to stop
+wasting my time, and move forward to hunt some commercial pray. I've visited
+surroundings stands to look for someone who would understand what are we doing,
+why, and be interested in cooperation. It was tough. Web Summit is large. I was
+told that there were over 70.000 attenders and 12.000 CEO. Founding someone
+who runs a REAL tech company suitable for us was a lot of walking and talking.
+Finally, I've succeeded but in a bit unexpected way.
+
+[![Vadim from Cervi Robotics](https://img.youtube.com/vi/iXUO7wY9EeM/0.jpg)](http://www.youtube.com/watch?v=iXUO7wY9EeM&t)
+
+Vadim has been a member of a [Pomeranian Export Broker](http://www.brokereksportowy.pl)
+team and it appeared that the darkest place is under the candlestick.
```

Comment:
@lwcislo I'm not sure how this can be interpreted, strategically this is questionable.

Reply:
@pietrushnic I tried to say, that a part of our own team was the best business opportunity, and looking far away wasn't so neccessary.